game/outfits.rpy

```diff
@@ -1921,6 +1939,15 @@ init -1 python in jn_outfits:
         accessory=get_wearable("jn_accessory_hairband_red"),
         hairstyle=get_wearable("jn_hair_twin_buns")
     ))
+    __register_outfit(JNOutfit(
+        reference_name="jn_office_outfit",
+        display_name="Office outfit",
+        unlocked=False,
+        is_jn_outfit=True,
+        clothes=get_wearable("jn_clothes_office_blazer"),
+        hairstyle=get_wearable("jn_hair_wavy"),
+        necklace=get_wearable("jn_necklace_necktie")
```

Comment:
Just a note to say that if we change the above for necktie, we'd need to update this also.